recipes/ostaphep/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "OstapHEP" %}
+{% set version = "1.3.3.0" %}
+{% set hash = "14447989be2ef14d45db292c41db8b18eaf07797ba4812d2775d75835ba90515" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/OstapHEP/ostap/archive/{{ version }}.tar.gz
+  sha256: {{ hash }}
+  folder: ostap
+
+build:
+  number: 0
+  skip: True  # [not linux]  
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - root >=6.16.0
+    - python
+    - ipython
+  run:
+    - python 
+    - root 
+    - future
+    - ipython
```

Comment:
```suggestion
    - ipython
    - gsl
```

Reply:
I've added gsl. But I've checked that root on conda-forge already require gsl. So I supposed that gsl would be  installed for root.

Reply:
It would and that’s why it currently works though it’s better to be explicit as ostap would suddenly break if ROOT stopped depending on GSL. Admittedly it’s unlikely in this case but it’s still nice to be complete.

More importantly there would be issues with migrating to new ABIs using the bots.

Reply:
Thanks, now it's clear.